cmd/execution/main.go

```diff
@@ -134,6 +136,7 @@ func main() {
 			flags.BoolVar(&pauseExecution, "pause-execution", false, "pause the execution. when set to true, no block will be executed, but still be able to serve queries")
 			flags.BoolVar(&enableBlockDataUpload, "enable-blockdata-upload", false, "enable uploading block data to GCP Bucket")
 			flags.StringVar(&gcpBucketName, "gcp-bucket-name", "", "GCP Bucket name for block data uploader")
+			flags.BoolVar(&atreeValidationEnabled, "atree-validation", false, "validates all atree values after mutations")
```

Comment:
@zhangchiqing @Kay-Zee We want to have this sanity check enabled on Canary, and turned off on Testnet and Mainnet

Reply:
ok